routes/reps.js

```diff
@@ -3,18 +3,10 @@ const express = require('express');
 const router = express.Router();
 const repsCtrl = require('../controllers/reps');
 
-router.get("/", function (req, res) {
-    res.send("index", {
-      user: req.user,
-    });
-  });
 
-  router.post('/', function (req, res) {
-    res.send("index", {
-      user: req.user,
-    });
-  }); 
-   
+
+// GET and POST will go here
+router.get('/', repsCtrl.index); 
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Nlopez4/fitness_tracking_app/pullRequest?prid=7840230)
```suggestion
router.get("/", repsCtrl.index); 
```